storyboard-text/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
     id("org.jetbrains.compose")
     id("com.vanniktech.maven.publish")
 
-    id("com.strumenta.antlr-kotlin") version "1.0.2"
+    id("com.strumenta.antlr-kotlin") version "1.0.5"
```

Comment:
Did this happen to fix the annoying bug where the generated file wouldn't compile because it used something like `isEmpty` instead of `isEmpty()`?! Because then we can hook generation into the actual build and avoid checking in the lexer/parser. Finally.